enet.pyx

```diff
@@ -819,6 +852,8 @@ cdef class Host:
 
     cdef ENetHost *_enet_host
     cdef bool dealloc
+    cdef object _interceptCallback
+    cdef object __weakref__
```

Comment:
This is for a weak reference for Host_instances_static variable. I'll try removing it and seeing whether the weak references dictionary will still exists.

We need a weak reference dictionary to map all instances of ENetHost to pyenet Host instances. Could have been done with a normal dictionary, but then there are deallocation problems (I.e. without weak ref dict the Host instances wouldn't be garbage collected)